### DISCUSSION TITLE
[admob] even TestIds returns the error: "The ad request was successful, but no ad was returned due to lack of ad inventory."

### DISCUSSION THREAD

**Discussion by alan-dx** [invertase/react-native-firebase#4809] [Q&A]
Hey guys, i'm creating an application and I'm using react-native-firebase / admob to display ads. However, even if I use the test settings with test unitId correctly, I am getting this error:

**Error: [admob / error-code-no-fill] The ad request was successful, but no ad was returned due to lack of ad inventory.**

Code implementation:
`                <BannerAd
                    unitId={TestIds.BANNER}
                    size={BannerAdSize.FULL_BANNER}
                    onAdFailedToLoad={(err) => {
                        console.log(err)
                    }}
                />`

- I have already checked all the settings and they were all done exactly as indicated in the documentation.
- I've tried using my Google AdMob id, but the error persists

Any idea what's going on?

**Comment by mikehardy** [comment]
This will be project specific according to my quick googling. There is something wrong with your account (for non test ads) or perhaps your package has been blacklisted or restricted in some way (for test ads)

**Reply by alan-dx** [reply]
> This will be project specific according to my quick googling. There is something wrong with your account (for non test ads) or perhaps your package has been blacklisted or restricted in some way (for test ads)

my admob account was created recently, researching I found that sometimes it takes some time for the ads to appear. In the case of test ads, I can't imagine what might be causing this restriction, any idea what it might be?
